geant4.sh

```diff
@@ -1,7 +1,7 @@
 package: GEANT4
 version: "%(tag_basename)s"
-tag: v4.10.01.p03-alice1
-source: https://github.com/alisw/geant4
+tag: v10.3.3
```

Comment:
this is quite a big version change for Run2. Is this supported by the DPG? Will they have to rerun all the Geant4 validation productions? Do we have a validation procedure when bumping Geant4?
@alisw/dpg

Reply:
IIRC it was discussed a couple of weeks ago at the CB, @pzhristov can maybe confirm that we've even taken a decision to this respect? I let him comment on this change.

Reply:
Yes, since Geant4 is not in production anyway, we better align all the versions. This will help to make also the comparisons in a meaningful way avoiding differences from the compiler.